schema/types/conversion_mechanisms/SAFEConversionMechanism.schema.json

```diff
@@ -28,9 +28,17 @@
     "conversion_mfn": {
       "description": "Is this an MFN flavored SAFE?",
       "type": "boolean"
+    },
+    "conversion_flavor": {
```

Comment:
@JSv4 Flavor seems an odd choice of words here. Feels too much like idiomatic US English to me. Perhaps we just call it `conversion_timing` to mirror the type?

Reply:
Haha, fair. I went back and forth on that. Give me a sec.

Reply:
Revised, @pjohnmeyer